test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Aggregation/AggregationController.cs

```diff
@@ -87,6 +87,11 @@ public virtual bool CleanCommands()
         {
             return true;
         }
+
+        public void Dispose()
+        {
+           // _db.Dispose();
```

Comment:
Once this is not used then better to remove disposable interface

Reply:
Hmm... yes.

---
In reply to: [352288339](https://github.com/OData/WebApi/pull/1963#discussion_r352288339) [](ancestors = 352288339)

Reply:
Added the interface because CA was complaining that the controller was not disposable.  However, if we actually call dispose on the context here, the context gets disposed after the controller returns but before serialization, so we get a null ref exception during serialization.

---
In reply to: [356372476](https://github.com/OData/WebApi/pull/1963#discussion_r356372476) [](ancestors = 356372476,352288339)